cloudpickle/cloudpickle.py

```diff
@@ -64,7 +64,7 @@
 
 
 if sys.version < '3':
-    from pickle import Pickler
+    from pickle import Pickler as _Pickler
```

Comment:
If we go with this, we better leave a comment as to _why_ we're using `_Pickler` here as some future contributor & maintainer may not know the context and end up breaking your use case..

Reply:
Very good point.

Reply:
@dbczumar -- could you add the context around this PR to the import(s) to help our future selves?